packages/mask/src/plugins/Pets/SNSAdaptor/PreviewBox.tsx

```diff
@@ -0,0 +1,151 @@
+import { makeStyles, useStylesExtends } from '@masknet/theme'
+import { Typography } from '@mui/material'
+import classNames from 'classnames'
+import { useI18N } from '../../../utils'
+import type { OwnerERC721TokenInfo } from '../types'
+import ModelView from './ModelView'
+
+export const useStyles = makeStyles()((theme) => ({
+    box: {
+        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: 'calc(100% - 16px)',
+        border: '1px dashed #bbb',
+        borderRadius: '4px',
+        marginTop: '16px',
+        boxSizing: 'border-box',
+        padding: '8px',
+    },
+    msgBox: {
+        width: '80%',
+        backgroundColor: '#fff',
+        borderRadius: '12px',
+        boxShadow: '0 0 8px #ddd',
+        opacity: 1,
+        pointerEvents: 'none',
+        transition: 'all 200ms',
+        padding: '12px',
+        fontSize: '12px',
+        lineHeight: '16px',
+        color: '#222',
+        textAlign: 'center',
+        marginBottom: '12px',
+        '&:before': {
```

Comment:
one colon for pseudo class, two colons for pseudo-elements
```suggestion
        '&::before': {
```